neard/src/cli.rs

```diff
@@ -161,6 +164,9 @@ struct NeardOpts {
     /// Whether the log needs to be colored.
     #[clap(long, arg_enum, default_value = "auto")]
     pub color: ColorOutput,
+    /// Enables export of span data using opentelemetry protocol.
+    #[clap(long)]
+    pub opentelemetry: bool,
```

Comment:
Rather than listing all these options inline, please consider adding a `struct O11yOpts` to the `o11y` crate and “merge” it with other options with `#[clap(flatten)]` & then passing in `O11yOpts` as a parameter to the `default_subscriber` function.

Reply:
Done